src/WebJobs.Script/Rpc/LanguageWorkerConstants.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public static class LanguageWorkerConstants
+    {
+        public const string FunctionWorkerRuntimeSettingName = "FUNCTIONS_WORKER_RUNTIME";
+        public const string DotNetLanguageWorkerName = "dotnet";
+        public const string NodeLanguageWorkerName = "node";
+        public const string JavaLanguageWrokerName = "java";
+        public const string DefaultWorkersDirectoryName = "workers";
+        public const string WorkersDirectorySectionName = "workersDirectory";
+        public const string WorkerConfigFileName = "worker.config.json";
+        public const string LanguageWorkerSectionName = "languageWorker";
+        public const string WorkerDescriptionLanguage = "language";
```

Comment:
Apply the "Name" suffix to all of these to continue the pattern above

Reply:
Added comments. Few constants are section names, rest of them are not.